main.py

```diff
@@ -61,8 +69,6 @@ def run_many(times):
 
 
 def shutdown():
-    print_markdown("## Clearing temp files")
-    cleanup()
```

Comment:
Why was this removed?

Reply:
Because if the shutdown action was called before the reddit id was determined, it deletes the whole temp folder.

Reply:
I could make it that it would remove the whole temp folder with all the sessions but that could affect current other sessions.

Reply:
> Because if the shutdown action was called before the reddit id was determined, it deletes the whole temp folder.

Could add a check to see if id is null, if it is, don't remove anything

Reply:
Good idea! Haven't thought about that.I'm working on something else now, in a few hours I'll update it.